Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6
+FROM python:3.7
```

Comment:
Do we need to upgrade the conformance dockerfile to use 3.7 as well?
https://github.com/Duke-GCB/calrissian/blob/2ed785497852f144da8236caccfedad556802d98/conformance/Dockerfile.conformance#L1

Reply:
Good catch, probably should do that